README.md

```diff
@@ -99,6 +99,17 @@ An example `finch.yaml` looks like this:
 cpus: 4
 # Memory: the amount of memory to dedicate to the virtual machine. (required)
 memory: 4GiB
+# CredsHelper: a list of credential helpers that will be installed and configured automatically. 
+# Currently,  ecr-login is the only supported option.
+# Once the option has been set the credential helper will be installed on either finch vm init or finch vm start. 
+# The binary will be downloaded on the host machine and a config.json will be created and populated inside the ~/.finch/ folder 
+# if it doesn't already exist. If it already exists, the value of credsStore will be overwritten. 
+# To opt out of using the credential helper, remove the value from the credsStore parameter of config.json 
+# and remove the creds_helper value from finch.yaml. 
+# To completely remove the credential helper, either remove the binary from ~/.finch/creds-helpers or remove the creds-helpers
+# folder entirely. (optional)
```

Comment:
This still didn't clarify how to use the ecr credential helper. Maybe add something like "supported credential helper list: ecr-login" and put the ecr credential helper link here?